XCUITests/FxScreenGraph.swift

```diff
@@ -0,0 +1,181 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import XCTest
+
+let SettingsScreen = "SettingsScreen"
+let TabTray = "TabTray"
+let TabTrayMenu = "TabTrayMenu"
+let BrowserTab = "BrowserTab"
+let BrowserTabMenu = "BrowserTabMenu"
+let BrowserTabMenu2 = "BrowserTabMenu2"
+let FirstRun = "OptionalFirstRun"
+let HomePageSettings = "HomePageSettings"
+let PasscodeSettings = "PasscodeSettings"
+let PasscodeIntervalSettings = "PasscodeIntervalSettings"
+let SearchSettings = "SearchSettings"
+let NewTabSettings = "NewTabSettings"
+let ClearPrivateDataSettings = "ClearPrivateDataSettings"
+let LoginsSettings = "LoginsSettings"
+let OpenWithSettings = "OpenWithSettings"
+let NewTabScreen = "NewTabScreen"
+let NewTabMenu = "NewTabMenu"
+
+let allSettingsScreens = [
+    SettingsScreen,
+    HomePageSettings,
+    PasscodeSettings,
+    SearchSettings,
+    NewTabSettings,
+    ClearPrivateDataSettings,
+    LoginsSettings,
+    OpenWithSettings,
+]
+
+func createScreenGraph(app: XCUIApplication, url: String = "https://www.mozilla.org/en-US/book/") -> ScreenGraph {
+    let map = ScreenGraph(app)
+
+    map.createScene(FirstRun) { scene in
+        scene.gesture(to: NewTabScreen) {
+            let firstRunUI = app.buttons["Start Browsing"]
+            if (firstRunUI.exists) {
```

Comment:
Swift (and gameplaykit) novice question here: in case if those elements like firstRunUI doesn't exist, does it throw an exception, or does it silently fail? (same for other tap() calls without checks) 

If it does silently fail, it would be nice if there is a blanket throw call or checker methods that we can use, but I guess we don't need it right away- we can add that feature later.  (same comment for tap() and other interactionmethods for ScreenGraphNode)